src/components/pages/public/index.js

```diff
@@ -4,6 +4,7 @@ import { Route, Switch, Redirect } from 'react-router-dom';
 import SignIn from './sign-in';
 import SignUp from './sign-up';
 import ForgotPassword from './forgot-password';
+import ResetPasswordConfirmation from '../../experiences/reset-password-confirmation';
```

Comment:
if you look here all page for this router all in the same folder (under public folder), they are accessed with ./ not from experiences